### PR TITLE
add readme, contributing, release, changelog files

### PR DESCRIPTION
**Pull request by jennyf19**
Not sure if we want this type of README here, i just copy/pasted the one we currently have for MS Identity Web, but this can be a place holder for now and I can delete the other one. thoughts?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +1,397 @@
...
-This project welcomes contributions and suggestions.  Most contributions require you to agree to a
-Contributor License Agreement (CLA) declaring that you have the right to, and actually do, grant us
-the rights to use your contribution. For details, visit https://cla.opensource.microsoft.com.
+This library contains a set of reusable classes useful in ASP.NET Core:
 
-When you submit a pull request, a CLA bot will automatically determine whether you need to provide
-a CLA and decorate the PR appropriately (e.g., status check, comment). Simply follow the instructions
-provided by the bot. You will only need to do this once across all repos using our CLA.
+- [Web applications](#web-apps) that sign in users, and, optionally, call web APIs
+- [Protected web APIs](#web-apis), which, optionally call downstream web APIs
```
Comment (pmaytak) [inline README.md]: Broken links for me.

**Comment by jennyf19** [comment]
yes...we have to fix the links, etc...but thought it would be more of a placeholder and then JM can update things

---
In reply to: [361693516](https://github.com/AzureAD/microsoft-identity-web/pull/29#issuecomment-361693516) [](ancestors = 361693516)

File: README.md [hunk 2]
```diff
@@ -1,14 +1,397 @@
+# Microsoft Identity Web
 
-# Contributing
+[![Build status](https://identitydivision.visualstudio.com/IDDP/_apis/build/status/AAD%20Samples/.NET%20client%20samples/ASP.NET%20Core%20Web%20App%20tutorial)](https://identitydivision.visualstudio.com/IDDP/_build/latest?definitionId=819)
```
Comment (jmprieur) [inline README.md]: I guess we need to change this one later when we have the CI build?

File: README.md (hunk 1 above)
Comment (jmprieur) [reply README.md]: I just checked #web-apis should work - when in the branch?

File: README.md [hunk 3]
```diff
@@ -1,14 +1,397 @@
...
-This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/).
-For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or
-contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
+to enable them to work with the Microsoft identity platform (formerly named Azure AD v2.0 endpoint). In the library, web apps and protected web APIs are collectively referred to as web resources.
+
+
+ ## Breaking changes
+
+ If you've been using Microsoft.Identity.Web in your project you might want to know that, on 12/05/2019, a number of APIs are renamed to be consistent:
+.AddMicrosoftIdentityPlatformAuthentication => AddSignIn
```
Comment (jmprieur) [inline README.md]: But we have added obsolete attributes. Might be worth looking at all the obsolete attributes

File: README.md [hunk 4]
```diff
@@ -1,14 +1,397 @@
...
+```CSharp
+using Microsoft.Identity.Web;
+
+public class Startup
+{
+  ...
+  public void ConfigureServices(IServiceCollection services)
+  {
+   ...
+   services.AddSignIn(Configuration);
```
Comment (jmprieur) [inline README.md]: Chris Ross recommended we also add the `services.AddAuthentication(AzureADDefaults.AuthenticationScheme)` before. Something to think of

**Comment by jennyf19** [comment]
yes. understood. i will make a separate issue for the README updates and copy over the comments from you and @pmaytak 

---
In reply to: [362093050](https://github.com/AzureAD/microsoft-identity-web/pull/29#issuecomment-362093050) [](ancestors = 362093050)

**Pull request merged**